main.go

```diff
@@ -173,22 +173,31 @@ func (f *FatalWriter) Write(p []byte) (n int, err error) {
 	return len(p), nil
 }
 
-func createLogConfig(context *cli.Context) logs.Config {
-	logFilePath := context.GlobalString("log")
-	logPipeFd := 0
-	if logFilePath == "" {
-		logPipeFd = 2
+func configLogrus(context *cli.Context) error {
+	if context.GlobalBool("debug") {
+		logrus.SetLevel(logrus.DebugLevel)
+		logrus.SetReportCaller(true)
 	}
-	config := logs.Config{
-		LogPipeFd:   logPipeFd,
-		LogLevel:    logrus.InfoLevel,
-		LogFilePath: logFilePath,
-		LogFormat:   context.GlobalString("log-format"),
-		LogCaller:   context.GlobalBool("debug"),
+
+	fmt := context.GlobalString("log-format")
+	switch fmt {
```

Comment:
nit: variable name clashes with the `fmt` package. Could do;

```patch
  	switch f := context.GlobalString("log-format"); f {
```

Reply:
done, thanks!